.github/workflows/cpu-inference.yml

```diff
@@ -15,7 +15,7 @@ concurrency:
 
 jobs:
   unit-tests:
-    runs-on: ubuntu-20.04
```

Comment:
FYI @mrwyattii - see the comment on the PR for the explanation of why.

Reply:
@loadams @delock we have self-hosted CPU runners. Can we please use those instead to avoid wasting GPU resources?

Use `runs-on: [self-hosted, cpu]`

Reply:
Thanks for the suggestion, updated.